config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  
+  get '/guests/:id', to: 'guests#show'
+  get '/rooms', to: 'rooms#index'
+
```

Comment:
We'll want a route here for _creating_ a new relationship between Guest and Room, from user story 2: 

```
post "/guests/:id/rooms", to: "guest_rooms#create"
```
This is one way to do it, but we want to make sure the `create` action matches the `post` verb.